libafl_qemu/src/qemu/config.rs

```diff
@@ -304,14 +297,16 @@ impl<R: AsRef<Path>> From<R> for Program {
 }
 
 #[derive(Debug, Clone, libafl_derive::Display, TypedBuilder, Getters)]
-#[builder(build_method(into = Result<Qemu, QemuInitError>), builder_method(vis = "pub(crate)",
+#[builder(builder_method(
+    vis = "pub(crate)",
```

Comment:
this needs to become `pub` (we can entirely remove the `vis` attribute since `pub` is the default)